examples/mspaint.py

```diff
@@ -8,15 +8,36 @@
 The example shows how to work with MS Paint application. It opens
 JPEG image and resizes it using "Resize and Skew" dialog.
 """
-
+import logging
+import sys
+from pywinauto import actionlogger
 from pywinauto import Application
 
+is_logging_enabled = False
+file_name_log = ''
+if len(sys.argv) >= 2 and len(sys.argv) <= 3:
+    if sys.argv[1] == '--log':
+        is_logging_enabled = True
+        file_name_log = 'log_mspaint.txt'
+        if len(sys.argv) == 3:
+            file_name_log = sys.argv[2]
+    else:
+        print("""Usage: python mspaint.py --log <filename>\n
+             --log - enable logging to a file (an optional parameter)\n
+             <filename> - log file name (an optional parameter, default: """)
+        exit()
```

Comment:
Default behavior should just print all the data to stdout instead of specified file. Why exit?